spec/factories/state_file_az_intakes.rb

```diff
@@ -127,6 +127,9 @@
     end
 
     trait :with_spouse do
+      raw_direct_file_data { StateFile::DirectFileApiResponseSampleService.new.read_xml('az_johnny_mfj') }
+      raw_direct_file_intake_data { StateFile::DirectFileApiResponseSampleService.new.read_json('az_johnny_mfj') }
+
       filing_status { 'married_filing_jointly' }
       spouse_first_name { "Susie" }
```

Comment:
Should we just remove these and keep the spouse name on `az_johnny_mfj` json since we are using the SSN from there too? Moira OHara [dust]

Reply:
oh good point